packages/manager/apps/dedicated/client/app/account/identity-documents/translations/Messages_fr_FR.json

```diff
@@ -16,12 +16,12 @@
   "user_account_identity_documents_selection_file_format": "Format: jpg, jpeg, pdf, png Taille maximale: 10 mo.",
   "user_account_identity_documents_selection_file_format_invalid": "Format de fichier non pris en charge. Veuillez utiliser l'un des formats acceptés : jpg, jpeg, pdf et png",
   "user_account_identity_documents_submit": "Envoyer mes documents",
-  "user_account_identity_documents_submit_success_message_title": "Merci, nous avons bien reçu vos documents!",
-  "user_account_identity_documents_submit_success_message_description": "Nous avons bien reçu l’ensemble de vos documents. Ils seront soumis à validation auprès de nos équipes. Vous serez informé par email de leur validation ou vous serez contacté si nous avons besoin d’informations complémentaires.",
-  "user_account_identity_documents_submit_error_message_description": "Vos documents n’ont pas été correctement envoyés. Veuillez refaire l’upload et l’envoi de vos documents",
+  "user_account_identity_documents_submit_success_message_title": "Merci, nous avons bien reçu vos documents !",
```

Comment:
**question:** Is it requested to add this extra space? If not, can you remove it please?

Reply:
Accordingly to @dectotam, normally in French it should have a space here.